src/benchmarks/micro/Program.cs

```diff
@@ -18,24 +14,9 @@ class Program
         static int Main(string[] args)
             => BenchmarkSwitcher
                 .FromAssembly(typeof(Program).Assembly)
-                .Run(args, GetConfig())
+                .Run(args, RecommendedConfig.Create(
+                    artifactsPath: new DirectoryInfo(Path.Combine(Path.GetDirectoryName(typeof(Program).Assembly.Location), "BenchmarkDotNet.Artifacts")), 
```

Comment:
This should probably be moved to the `RecommendedConfig` instead.

Reply:
the problem is that we want `typeof(Program).Assembly.Location` which is different than `BenchmarkDotNet.Extensions.dll` location

We could use `Assembly.GetEntryAssembly` or ``Assembly.GetExecutingAssembly` but I have bad experience using those methods (they have platform specific behaviour depending on what has actually run the benchmarks) and AFAIR they are not implemented on CoreRT (which I hope to add sooon)